docs/displaysystem/angular.md

```diff
@@ -88,11 +82,130 @@ z.angularUnwatchGlobal(String name)
 
 ```
 
-Let's make an button, that increment 'run' variable by 1 when it is clicked.
-z.angularBind("run", 0) will initialize 'run' to zero. And then register watcher of 'run'.
+Let's make a button. When it is clicked, the value of `run` will be increased 1 by 1.
+
+<img src="/assets/themes/zeppelin/img/screenshots/display_angular2.png" width="60%" />
+
+`z.angularBind("run", 0)` will initialize `run` to zero. And then, it will be also applied to `run` in `z.angularWatch()`.
+When the button is clicked, you'll see both `run` and `numWatched` are incremented by 1.
+
+<img src="/assets/themes/zeppelin/img/screenshots/display_angular3.png" width="60%" />
+
+## Let's make it Simpler and more Intuitive
+In this section, we will introduce a simpler and more intuitive way of using **Angular Display System** in Zeppelin. 
+
+### How can we use it?
+Here are some usages. 
+
+#### Import 
+
+```scala
+import org.apache.zeppelin.display._
+import AngularElem._
+```
+
+#### Display Element
+```scala
+// automatically convert to string and print with %angular display system directive in front.
+<div><div>.display
+```
+#### Event Handler
+```scala
+// on click
+<div></div>.onClick(() => {
+   my callback routine
+}).display
+
+// on change
+<div></div>.onChange(() => {
+  my callback routine
+}).display
+
+// arbitrary event
+<div></div>.onEvent("ng-click", () => {
+  my callback routine
+}).display
+```
+
+#### Bind Model
+```scala
+// bind model
+<div></div>.model("myModel").display
+
+// bind model with initial value
+<div></div>.model("myModel", initialValue).display 
+```
+
+#### Interact with Model
+```scala 
+// read model
+AngularModel("myModel")()
+
+// update model
+AngularModel("myModel", "newValue")
+```
+
+<br/>
+### Example: Basic Usage
+Using the above basic usages, you can apply them like below examples. 
+
+#### Display Elements
+
+```scala
+<div style="color:blue">
+  <h4>Hello Angular Display System</h4>
+</div>.display
+```
+
+#### OnClick Event
+```scala
+<div class="btn btn-success">
+  Click me
+</div>.onClick{() =>
+  // callback for button click
+}.display
+```
+
+#### Bind Model
+
+{% raw %}
```

Comment:
That's for escaping in Jekyll
http://blog.slaks.net/2013-06-10/jekyll-endraw-in-code/
probably because of the `{{{{title}}}}` in the doc